Doc/library/functions.rst

```diff
@@ -496,7 +496,7 @@ are always available.  They are listed here in alphabetical order.
               n += 1
 
 
-.. function:: eval(expression[, globals[, locals]])
+.. function:: eval(expression[, globals[, locals]], *, closure=None)
```

Comment:
This is the docs for `eval`, but you actually changed only `exec`.

Reply:
Oops.  Fixed.

I put the description of the *closure* argument correctly in the docs for `exec`, I just modified the wrong prototype.  (I kept making this dumb mistake--originally the branch was called `eval_closure`.)